app/vmalert/remotewrite/remotewrite.go

```diff
@@ -138,14 +142,11 @@ func (c *Client) Close() error {
 
 func (c *Client) run(ctx context.Context) {
 	ticker := time.NewTicker(c.flushInterval)
-	wr := prompbmarshal.WriteRequest{}
+	wr := &prompbmarshal.WriteRequest{}
 	shutdown := func() {
 		for ts := range c.input {
 			wr.Timeseries = append(wr.Timeseries, ts)
 		}
-		if len(wr.Timeseries) < 1 {
```

Comment:
Why this condition has been removed?

Reply:
`flush` already checks `wr.Timeseries` length as a first step. Removed the check here only to simplify flow